README.md

```diff
@@ -237,3 +239,7 @@ This feature relies on a defined schema in a YML file. It extracts data from bot
 ### <a id="logtailer">dbt logs force tailing</a>
 
 ![dbt logs](./media/images/dbt-log.gif)
+
+### <a id="projectscan">Altimate scan</a>
```

Comment:
We should some details about the scans performed and what each of the scan mean

Reply:
done